docs/satellite-clone.md

```diff
@@ -91,3 +91,24 @@ On the target server:
       # ansible-playbook satellite-clone-playbook.yml
     ```
   The playbook run may may take a while to complete.
+
+## Cloning a Satellite With Remote Databases
+
+Note: the backup used for this procedures must have the correct options to manage remote databases in `/etc/foreman-installer/scenarios.d/satellite-answers.yaml`. Please check that the answers file has the correct database management values filled configuration backups.
```

Comment:
Can you elaborate on what the user actually needs to check here?

Reply:
ya, I can specify the specific ones. Unfortunately the confusion is the result of a docs bug: https://bugzilla.redhat.com/show_bug.cgi?id=188784

Reply:
Updated, I mentioned the BZ and the specific options.